src/config/dynamic-theme-fixes.config

```diff
@@ -13253,14 +13253,23 @@ INVERT
 #cloudos-loading-spinner
 .editor-container
 .light > svg
-.apple-icloud-logo > g > path:first-child
-.apple-icloud-logo.dark > g > path:last-child
+#.apple-icloud-logo > g > path:first-child
```

Comment:
Hi, could you please tell about the `#` symbol? If you wanted to disable these lines, you had to simply remove them. The `#abc` is an `id="abc"` selector in CSS, but together with a dot it produces an invalid selector.

Reply:
oh sorry my bad 😓, you're right I should've just removed the lines. I didn't think about the id selector at the time, thought I would comment it out in case it may be needed in the future, but I don't think it will.

Reply:
but it seems to support comments. just tried with the dev tools.
just had to use // instead of # 

(I was at the same time working on configuration stuff and there is # used as a comment, so I mixed it up 😅)

<img width="832" alt="image" src="https://github.com/user-attachments/assets/8cb40037-2d29-4b8c-8f6e-1d8ee4d14a9e">